lib/rules/disallow-space-after-object-keys.js

```diff
@@ -134,7 +134,7 @@ module.exports.prototype = {
             var maxKeyEndPos = 0;
             var tokens = [];
             node.properties.forEach(function(property) {
-                if (property.shorthand || property.kind !== 'init') {
+                if (property.shorthand || property.kind !== 'init' || property._babelType === 'SpreadProperty') {
```

Comment:
1. Under which circumstances was the type being changed within acorn to esprima?
2. Why was the original type not preserved as `type` and a babel-specific type kept private?